config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,16 @@
+require 'okcomputer'
+
```

Comment:
I've been mounting okcomputer at "status" but obviously it's a personal preference thing:

```
# /status for 'upness', e.g. for load balancer
# /status/all to show all dependencies
# /status/<name-of-check> for a specific check (e.g. for nagios warning)
OkComputer.mount_at = 'status'
OkComputer.check_in_parallel = true
```